src/EditorFeatures/Core/Tagging/TaggerContext.cs

```diff
@@ -86,11 +87,15 @@ public void SetSpansTagged(IEnumerable<DocumentSnapshotSpan> spansTagged)
             this._spansTagged = spansTagged ?? throw new ArgumentNullException(nameof(spansTagged));
         }
 
-        public IEnumerable<ITagSpan<TTag>> GetExistingTags(SnapshotSpan span)
+        public IEnumerable<ITagSpan<TTag>> GetExistingContainingTags(SnapshotPoint point)
         {
-            return _existingTags != null && _existingTags.TryGetValue(span.Snapshot.TextBuffer, out var tree)
-                ? tree.GetIntersectingSpans(span)
-                : SpecializedCollections.EmptyEnumerable<ITagSpan<TTag>>();
+            if (_existingTags != null && _existingTags.TryGetValue(point.Snapshot.TextBuffer, out var tree))
+            {
+                return tree.GetIntersectingSpans(new SnapshotSpan(point.Snapshot, new Span(point, 0)))
+                           .Where(s => s.Span.Contains(point));
+            }
+
+            return SpecializedCollections.EmptyEnumerable<ITagSpan<TTag>>();
```

Comment:
this change was made due to some issue with highlight-refs when you have two things right next to each other that need highlighting.  This doesn't really happen in C#/VB (as there is always punctuation or spaces between symbols), but happens in regex where you can have multiple references abutting each other.